server/public/model/config.go

```diff
@@ -256,6 +256,8 @@ const (
 	OpenidSettingsDefaultScope = "profile openid email"
 
 	LocalModeSocketPath = "/var/tmp/mattermost_local.socket"
+
+	AlwaysShowTeamSidebar = ""
```

Comment:
Is this value ever referenced?

Reply:
+1

Reply:
Thanks for the catch. 
This value had no use. I removed it.